src/libraries/System.Data.Odbc/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,363 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.Data.Odbc.OdbcParameter.get_Offset</Target>
```

Comment:
These look wrong.  We shouldn't be missing members on runtime implementations.

Reply:
Yes, I wonder why these are missing on the RID specific assets?

Reply:
Looks to me like it is this: https://github.com/dotnet/runtime/blob/895c99cc502913fd7393b95af7cae5b9e6fbdea3/src/libraries/System.Data.Odbc/src/MatchingRefApiCompatBaseline.txt#L4-L6

Seems like that should be peeled off into a follow up bug.

Reply:
> Seems like that should be peeled off into a follow up bug. Those APIs should be representable in refs on the correct frameworks, and thus make it into the PNSE assemblies.

Agreed.